src/app/templates/scripts/_build-test.js

```diff
@@ -10,6 +10,8 @@ glob('test/**/*.test.js', (err, files) => {
   }
   browserify(files)
     .transform('babelify')
+    .transform('browserify-shim')
+    .transform('browserify-versionify')
```

Comment:
I think we should only have to do `browserify-shim` here as the package.json entry should handle the `browserify-versionify` for us.